docs/part-3-serve-and-deploy-the-model/chapter-13-deploy-and-access-the-model-on-kubernetes-with-bentoml.md

```diff
@@ -130,195 +134,180 @@ flowchart TB
 
 ## Steps
 
-### Create the Kubernetes cluster
-
-In order to deploy the model on Kubernetes, you will need a Kubernetes cluster.
-Follow the steps below to create one.
-
-=== ":simple-googlecloud: Google Cloud"
-
-    **Enable the Google Kubernetes Engine API**
-
-    You must enable the Google Kubernetes Engine API to create Kubernetes clusters
-    on Google Cloud.
-
-    [Enable Google Kubernetes Engine API :octicons-arrow-up-right-16:](https://console.cloud.google.com/flows/enableapi?apiid=container.googleapis.com){ .md-button .md-button--primary }
-
-    **Create the Kubernetes cluster**
-
-    Create the Google Kubernetes cluster with the Google Cloud CLI.
+### Create a BentoML "Bento"
+
+A "Bento" is a BentoML artifact that packages your model, code, and environment
+dependencies into a single file. It is the standard format for saving and
+sharing ML models.
+
+A Bento is described in a `bentofile.yaml` file. It contains the following
+information:
+
+- The service filename and class name
+- The Python packages required to run the service
+- The Docker configuration, such as the Python version to use
+
+Create a new `bentofile.yaml` file in the `src` directory with the following
+content:
+
+```yaml title="src/bentofile.yaml"
+service: 'serve:CelestialBodiesClassifierService'
+include:
+  - serve.py
+python:
+  packages:
+    - "tensorflow==2.12.0"
+    - "matplotlib==3.7.1"
+    - "pillow==10.2.0"
```

Comment:
In addition, I had to explictly pass pydantic v2 (`"pydantic==2.6.1"`) to avoid an error later when running the docker image (otherwise pydantic v1 is installed by default as dependency):

```
docker run --rm -p 3000:3000 celestial-bodies-classifier:latest
Error: [bentoml-cli] `serve` failed: Failed loading Bento from directory /home/bentoml/bento: Failed to import module "serve": The new SDK runs on pydantic>=2.0.0, but the you have 1.10.14. Please upgrade it.
```

Reply:
Good catch.